terraform/market_intelligence/README.md

```diff
@@ -0,0 +1,137 @@
+# Real-Time Anomaly Detection project deployment
```

Comment:
I think this title does not correspond to this guide. Please update this file.

Reply:
Removed the line and pushed the changes.